test/integration/ga.ts

```diff
@@ -59,10 +145,10 @@ describe('Generalized Account', function () {
 
   it('Fail on make GA on already GA', async () => {
     await aeSdk.createGeneralizedAccount('authorize', authContractSource)
-      .should.be.rejectedWith(`Account ${gaAccount.publicKey} is already GA`)
+      .should.be.rejectedWith(`Account ${gaAccount.publicKey.toString()} is already GA`)
```

Comment:
isn't it already a string?

Reply:
It's `string | Buffer`

Reply:
`generateKeyPair` definition is fixed now